charts/sourcegraph/values.yaml

```diff
@@ -191,23 +247,39 @@ frontend:
     PROMETHEUS_URL:
       value: http://prometheus:30090
   image:
+    # -- Docker image tag for the `frontend` image
     defaultTag: 3.37.0@sha256:724ccb31e95237cbe12b51f3c8b8d22a7d0ac65a73ee8084614360c2018ce3ca
+    # -- Docker image name for the `frontend` image
     name: "frontend"
   ingress:
+    # -- Enable ingress for the Sourcegraph server
+    enabled: true
+    # -- Annotations for the Sourcegraph server ingress. For example, securing ingress with TLS provided by [cert-manager](https://cert-manager.io/docs/usage/ingress/)
     annotations:
       kubernetes.io/ingress.class: nginx
       nginx.ingress.kubernetes.io/proxy-body-size: 150m
-    enabled: true
-    host: "" # External host name for SSL
-    ingressClassName: null # Available in Kubernetes 1.18+
-    tlsSecret: "" # Secret containing SSL cert
+    # -- External hostname for the Sourcegraph server ingress (SSL)
+    host: ""
+    # -- IngressClassName for the Ingress (Available in Kubernetes 1.18+)
+    ingressClassName: null
+    # -- Secret containing SSL cert
+    tlsSecret: ""
+  # -- Security context for the `frontend` container,
+  # learn more from the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container)
   containerSecurityContext:
     allowPrivilegeEscalation: false
     runAsUser: 100
     runAsGroup: 101
     readOnlyRootFilesystem: true
-  privileged: true # Creates Role instead of using existing roles
+  # -- Security context for the `frontend` pod,
+  # learn more from the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod)
+  podSecurityContext: {}
+  # -- Creates Role instead of using existing roles
```

Comment:
Uses `view` ClusterRole if set to false, there's no customization currently allowed

Reply:
Updated to 

```md
Enable creation of Role and RoleBinding (RBAC). Uses [view](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles) ClusterRole if set to false
```